app/controllers/treatments/massage_trims_controller.rb

```diff
@@ -0,0 +1,15 @@
+class Treatments::MassageTrimsController < Treatments::TreatmentsController
+
+  def new
+    @user = user
+    @treatment = @user.treatments.new
```

Comment:
Isn't '#build' aliased to '#new'?
https://github.com/rails/rails/blob/master/activerecord/lib/active_record/relation.rb#L124